src/styles/main.scss

```diff
@@ -9,6 +9,6 @@ body {
 * {
   box-sizing: border-box;
 }
-
+@import "./utils/variables";
 @import "_header.scss";
 @import "_catalog.scss";
```

Comment:
Everything's Great 😊

One small suggestion for naming and folder structures:

```
styles
: main.scss
-- utils
    : _variables.scss
    : _extends.scss (for global styles, like the ones you're applying on body or html tag)
-- blocks
    : catalog.scss
```

Reply:
Thank you for suggestions Saleh. I will keep your suggestions in mind for the future projects. 😊

Reply:
I'm happy to help